app/templates/client/dev/components/todo/todo.js

```diff
@@ -1,51 +1,75 @@
-;(function(angular)
-{
-  "use strict";
-
-  angular
-    .module('myAwesomeApp')
-    .controller('TodoController', ['$log', 'Todo', 'TodoDAO', function($log, Todo, TodoDAO)
-    {
-      var self = this;
-
-      self.todo = new Todo();
-      self.todos = [];
-
-      self.createTodo = function(todo)
-      {
-        var _onSuccess = function(newTodo)
-        {
-          self.todos.push(newTodo);
-          self.todo = new Todo();
-        };
+(function () {
```

Comment:
https://github.com/johnpapa/angular-styleguide/blob/master/README.md#style-y010, has a specified style for IIFE. Do we want to branch out from the style guide? I have never run into any issues when code like this.